holocube/element/cube.py

```diff
@@ -127,56 +110,53 @@ def init(cls, eltype, data, kdims, vdims):
 
 
     @classmethod
-    def validate(cls, columns):
+    def validate(cls, holocube):
         pass
 
 
     @classmethod
-    def values(cls, columns, dim, expanded=True, flat=True):
+    def values(cls, holocube, dim, expanded=True, flat=True):
```

Comment:
I might have left the name as ``columns`` though I don't mind ``holocube`` if you think it is clearer...